compiler/src/iree/compiler/Dialect/Stream/Transforms/Passes.cpp

```diff
@@ -28,6 +28,12 @@ static llvm::cl::opt<bool> clReplicateGlobalsPerAffinity(
         "Replicates globals for each unique affinity they are used with."),
     llvm::cl::init(false));
 
+static llvm::cl::opt<bool> clTransferGlobalsPerAffinity(
+    "iree-stream-experimental-transfer-globals-per-affinity",
```

Comment:
This should be a mode flag or enum - one flag for the feature, either it's off (not set) or one of N enum values passed as an option to the pass.